services/s3/src/main/java/software/amazon/awssdk/services/s3/endpoints/internal/Value.java

```diff
@@ -364,15 +381,15 @@ public static Endpoint fromNode(JsonNode node) {
             JsonNode propertiesNode = objNode.get(PROPERTIES);
             if (propertiesNode != null) {
                 propertiesNode.asObject()
-                    .forEach((k, v) -> {
-                        b.property(k, Value.fromNode(v));
-                    });
+                              .forEach((k, v) -> {
```

Comment:
Do we want indentation changes like these to go through ? Feels like we have a lot to indent if we start looking into our entire codebase. Would it be better to leave them as they are ? It also shows up in the "git blame" section

Reply:
It just did that since it was a copy+paste of code, which reformats things automatically in Intellij. I didn't do it intentionally. I can try to revert these types of changes. Hopefully I don't miss any.

Reply:
I actually don't need all of these files! I've deleted a lot of them.